chart/epinio/templates/dex.yaml

```diff
@@ -77,10 +77,12 @@ metadata:
   namespace: {{ .Release.Namespace }}
   annotations:
     cert-manager.io/cluster-issuer: {{ default .Values.global.tlsIssuer .Values.global.customTlsIssuer | quote }}
-    kubernetes.io/ingress.class: traefik
     traefik.ingress.kubernetes.io/router.entrypoints: websecure
     traefik.ingress.kubernetes.io/router.tls: "true"
 spec:
+  {{- if .Values.ingressClassName }}
+  ingressClassName: "{{ .Values.ingressClassName }}"
```

Comment:
```suggestion
  ingressClassName: "{{ .Values.ingress.ingressClassName }}"
```

Reply:
dtto

Reply:
:facepalm: Will be fixed.

Reply:
Done. Commit d9bc90a96661aae9568389e4d6dacc2227812047